privacy/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang=en>
+	<head>
+		<title>The Forum Helpers</title>
+		<link rel="shortcut icon" type="image/png" href="../resources/favicon.png">
+		<link rel="stylesheet" type="text/css" href="../resources/stylesheet.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="Description" content="The official website for the Scratch Forum Helpers. This contains information about who we are and what we do.">
+
+		<!-- Global site tag (gtag.js) - Google Analytics -->
+		<script async src="https://www.googletagmanager.com/gtag/js?id=G-FGN5P806VX"></script>
+		<script>
+			window.dataLayer = window.dataLayer || [];
+			function gtag(){dataLayer.push(arguments);}
+			gtag('js', new Date());
+
+			gtag('config', 'G-FGN5P806VX');
+		</script>
+	</head>
+
+	<body>
+		<header id="header"></header>
+
+		<div class="privacy_content">
+			<h1 style="text-align: center;">Privacy</h1>
+			<h3>Google Analytics</h3>
+			<ul>
+				<li>In order to gain statistics about the number of people who are using our site, and how people interact with the site, we use Google Analytics to track page views and interactions.</li>
+				<li>Google Analytics tracks your website usage by placing cookies on your computer and by continuing to use this site, you agree to the use of these cookies.</li>
+				<li>We recommend that you check out <a href="https://policies.google.com/technologies/partner-sites">How Google uses data when you use our partners' sites or apps</a> for additional information about Google Analytics.</li>
+				<li>Examples of data that Google Analytics may collect include: IP Address, Operating System, the amount of time that you spend on each page, and what you click while on the page.</li>
+				<li>This data is not shared or sold with anybody besides site developers, who use it to improve the site based on usage statistics.</li>
+				<li>If you do not agree to the use of Google Analytics, please discontinue your use of this site. Alternatively, consider installing the <a href="https://support.google.com/analytics/answer/181881?hl=en-GB">Google Analytics opt-out browser add-on</a>.</li>
+			</ul>
+
+			<h3>Other</h3>
+			<ul>
+				<li>Aside from Google Analytics, this site also stores two cookies on your computer which save whether or not you have agreed to the use of Google Analytics, and the date that you agreed.</li>
+				<li>This cookie expires each month, so you will have to re-accept the use of Google Analytics when you visit our site in a month that you have not yet accepted.</li>
+				<li>Questions? Feel free to contact us by opening a discussion on our GitHub repository.</li>
```

Comment:
Maybe add a link to the Github repo? I guess there's always a link in the footer, but..

Reply:
Other than that, looks great.

Reply:
Yep, I can add a link!